assets/helpers/user/user.js

```diff
@@ -4,14 +4,21 @@
 
 import { routes } from 'helpers/routes';
 import * as cookie from 'helpers/cookie';
-import { getSession } from 'helpers/storage';
 import { get as getCookie } from 'helpers/cookie';
+import { getSession } from 'helpers/storage';
 import { defaultUserActionFunctions } from 'helpers/user/defaultUserActionFunctions';
 import type { UserSetStateActions } from 'helpers/user/userActions';
 
 
 // ----- Functions ----- //
 
+function isTestUser(): boolean {
+  const isDefined = x => x !== null && x !== undefined;
```

Comment:
Could we add a type to x? I think it looks a little neater:

`const isDefined = (x: string) => x !== null && x !== undefined;`

Reply:
Argh, just merged! I'll change it for my next PR